tests/phpunit/Compat/WC_Stripe_Subscription_Initial_Test.php

```
@@ -0,0 +1,141 @@
+		$initial_order->set_payment_method( WC_Stripe_UPE_Payment_Gateway::ID );
+		$initial_order->save();
+
+		$order_helper->update_stripe_customer_id( $initial_order, $customer );
+		$initial_order->save_meta_data();
+
+		$_POST = [
+			'payment_method'           => 'stripe',
+			'wc-stripe-payment-method' => 'pm_test_123',
+		];
+
+		$pre_http_request_response_callback = function (
```

PHPMD (2.15.0)

[error] The variable $pre_http_request_response_callback is not named in camelCase. (undefined)

(CamelCaseVariableName)